packages/theming/apple-theme/src/appleTheme.macos.ts

```diff
@@ -7,7 +7,7 @@ export function appleSpacing(): Spacing {
   return {
     s2: '8px',
     s1: '12px',
-    m: '16px',
```

Comment:
This seems like a different change than the other ones - I think we should probably keep this named as "m" for now. Eventually we want to remove spacing altogether from the theme and all these will be removed.